packages/components/src/Toaster/Toaster.module.css

```
@@ -12,23 +12,25 @@
 	width: 100%;
 	opacity: 0;
-	transform: translateY( 100% );
-	transition: opacity 0.3s ease-in-out, transform 0.3s ease-in-out;
+	transform: translateY(100%);
+	transition:
+		opacity 0.3s ease-in-out,
+		transform 0.3s ease-in-out;
 }
 
-.toast-message :global( .components-snackbar ) {
+.toast-message :global(.components-snackbar) {
```

Stylelint (17.4.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)